man/fi_endpoint.3.md

```diff
@@ -1048,11 +1048,6 @@ transfer operation in order to guarantee that ordering is met.
   update operations.  If not atomic updates may be
   transmitted out of order from their submission.
 
-*FI_ORDER_NONE*
```

Comment:
Do you want to remove or add a deprecated note like the others?

Reply:
This one was missed in the transition. Need to add a deprecated note.